bindings/python/pyiree/compiler/compiler.cc

```diff
@@ -76,6 +76,7 @@ bool LLVMOnceInit() {
   // supported.
 #ifdef IREE_REGISTER_MLIR_PASSES
   mlir::registerMlirPasses();
+  mlir::iree_compiler::registerAllIreePasses();
```

Comment:
This shouldn't need to be in the ifdef, I think. It's only required because sometimes we link in random other stuff that registers MLIR passes in its own way.